docs/unstructured_quad_mesh/unstructured_quad_mesh.md

```diff
@@ -0,0 +1,128 @@
+# Unstructured quadrilateral mesh
+
+Herein we describe the conventions taken in the implementation for two-dimensional unstructured quadrilateral meshes. Principally, this relates to how a file with the extension `.mesh` encodes information about the numbering and orientation of elements in an unstructured quadrilateral mesh with possibly curved boundaries.
```

Comment:
aha, gotcha. I can try to trim down the discussion or separate lines more. I wasn't sure how thorough this description should be for a new user

Reply:
No, sorry, I don't mean to remove some of these sentences - the docs are really great! I just wanted to say that I would like to see a few "newline characters" inside these paragraphs. They won't show up in the rendered docs but make it easier to track changes in the markdown code.

Reply:
Instead of having
```
Herein we describe the conventions taken in the implementation for two-dimensional unstructured quadrilateral meshes. Principally, this relates to how a file with the extension `.mesh` encodes information about the numbering and orientation of elements in an unstructured quadrilateral mesh with possibly curved boundaries.
``` 
in one single line, I would propose to use multiple lines like 
```
Herein we describe the conventions taken in the implementation for two-dimensional 
unstructured quadrilateral meshes. Principally, this relates to how a file with 
the extension `.mesh` encodes information about the numbering and orientation of 
elements in an unstructured quadrilateral mesh with possibly curved boundaries.
```

Reply:
I second this proposal - it is seemingly trivial but makes reviewing changes much easier when viewing diffs directly in GitHub.

Reply:
aha that is all you meant. Groovy, that is an easy change I can make